packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2934,8 +2934,9 @@ Iterable<DiagnosticsNode> _describeRelevantUserCode(Element element) {
       final String? devToolsInspectorUri =
           WidgetInspectorService.instance._devToolsInspectorUriForElement(target);
       if (devToolsInspectorUri != null) {
-        devToolsDiagnostic = DiagnosticsNode.message(
+        devToolsDiagnostic = DevToolsDeepLinkDiagnosticsNode(
```

Comment:
consider moving some of the logic from _devToolsInspectorUriForElement into DevToolsDeepLink as it is tightly coupled and that would make the DevToolsDeepLink class a little less surprising than if its public constructor takes an arbitrary URL which might or might not be from DevTools.